xapian-letor/include/xapian-letor/scorer.h

```diff
@@ -65,7 +65,22 @@ class XAPIAN_VISIBILITY_DEFAULT NDCGScore: public Scorer {
     ~NDCGScore();
 
     double score(const std::vector<FeatureVector> & fvv) const;
+};
+
+/** ERRScore class
```

Comment:
"ERRScore class" is not a good brief description.  It's just the name of the class (which in this case doesn't really usefully explain itself, since it's not clear what "ERR" means) plus the word "class".

Something like "Expected Reciprocal Rank Scorer" would at least explain the acronym.